backend/cmd/main.go

```diff
@@ -2,25 +2,28 @@ package main
 
 import (
 	"github.com/gorilla/mux"
+	"github.com/kyma-incubator/Kyma-Showcase/internal/api"
+	database2 "github.com/kyma-incubator/Kyma-Showcase/internal/database"
```

Comment:
Why is this import alias as `database2`? Do you have any second accessor to `database`? You can shorten it into `db` maybe.